src/sparql/icd10who-relevant-diseases.sparql

```diff
@@ -33,5 +33,5 @@ WHERE {
       }
     }
   }
-  FILTER(isIRI(?term))
+  FILTER(isIRI(?term) && (regex(str(?term), "https://icd.who.int/browse10/2019/en#/")))
```

Comment:
this is what I did in my other PR, so this should not be a conflict when I merge.

Reply:
YOu did thi for -signature.sparql, not -disease!

Reply:
ahhh, I didn't see that